libs/common/src/auth/services/token.service.ts

```diff
@@ -314,12 +314,18 @@ export class TokenService implements TokenServiceAbstraction {
       );
     }
 
-    const decryptedAccessToken = await this.encryptService.decryptString(
-      encryptedAccessToken,
-      accessTokenKey,
-    );
-
-    return decryptedAccessToken;
+    try {
+      const decryptedAccessToken = await this.encryptService.decryptString(
+        encryptedAccessToken,
+        accessTokenKey,
+      );
+      return decryptedAccessToken;
+    } catch (e) {
+      // Note: This should be replaced by the owning team with appropriate, domain-specific behavior.
+      // eslint-disable-next-line no-console
+      console.error("[TokenService] Error decrypting access token", e);
+      throw e;
```

Comment:
```suggestion
      // Note: This should be replaced by the owning team with appropriate, domain-specific behavior.
      this.logService.error("[TokenService] Error decrypting access token", e);
      throw e;
```

Looks like logService is available here.